mmvb_backend/toy_ais/management/commands/register_ais.py

```diff
@@ -21,7 +24,7 @@ def handle(self, *args, **options):
                 # TODO: dynamically get server
                 AIImplementation.objects.create(
                     name=toy_ai.name,
-                    base_url=f"http://localhost:8000/toy_ais/{toy_ai.slug_name}",
```

Comment:
good catch – can you investigate how to do the environment bit? (not for this PR though)

Reply:
I'm not sure if I follow what you mean. What is the environment bit?

Reply:
[Line 10](https://github.com/FG-AI4H-TG-Symptom/fgai4h-tg-symptom-assessment-mmvb-backend/pull/54/files/38f302833eb363fdd51a983940e0925c84617268#diff-01f3a0125a55090d9a27352380d3a04aR10) in your changed version
but we might also just open an issue for that